nemo/nemo/utils/exp_logging.py

```diff
@@ -80,20 +83,43 @@ def __init__(
         # tb_dir behaves the same as ckpt_dir except default folder name is
         # tensorboard instead of checkpoints
         self.tb_dir = tb_dir
+        tm_suf = time.strftime('%Y-%m-%d_%H-%M-%S')
+        if local_rank is not None and add_time:
```

Comment:
Ok, is this the same local_rank as in NMF.__init__:
local_rank (int): Process rank. Should be set by distributed runner ?

I see that by default it is set to None. What does it mean? And what 0 mean? 

I am guessing now, but maybe those two in fact mean the same, i.e. main process in the case of single process/ main process in distributed training? Maybe those could be unified?

Reply:
None means single-gpu
A numerical value means multi-gpu or multi-node. Generally, you just want one runner to do things in a multi process run and we just use process id 0.

Reply:
I see, I think it should be explained properly in the code and added to documentation.

Reply:
Added to ExpManager docstring